app/src/main/kotlin/com/intive/picover/camera/repository/ImageFileRepository.kt

```diff
@@ -0,0 +1,20 @@
+package com.intive.picover.camera.repository
+
+import android.content.Context
+import android.net.Uri
+import android.os.Environment
+import androidx.core.content.FileProvider
+import dagger.hilt.android.qualifiers.ApplicationContext
+import java.io.File
+import javax.inject.Inject
+
+// TODO-BCH Should be unit tested after Kotest setup
+class ImageFileRepository @Inject constructor(
+	@ApplicationContext private val context: Context
+) {
+
+	fun createTempFileAndGetUri(): Uri? {
+		val tempFile = File.createTempFile("IMG_", ".jpg", context.getExternalFilesDir(Environment.DIRECTORY_PICTURES))
```

Comment:
it would be nice to extract these strings to vals or even create a file like Constants

Reply:
What do we gain by doing so? It's used only in one place

Reply:
Okey, maybe I am too much sensitive on this :D But I bet `.jpg` will be used in the future not only once ;p For now as it's used only here it can be as it is